docs/reference/tab-widgets/troubleshooting/data/start-ilm.asciidoc

```diff
@@ -61,14 +61,14 @@ The reponse will look like this:
 // end::cloud[]
 
 // tag::self-managed[]
-<<ilm-start, Start ILM>>:
+<<ilm-start, Start Index Lifecycle Management>>:
```

Comment:
I am curious why are we specifically typing here `Index Lifecycle Management` and not `{ilm}` like we do in the `start-slm` doc?

Reply:
I see, that's why in the slm it is `<<ilm-start, Start>> {slm}`. I would suggest to do the same here so we are consistently using the macros.